README.md

```diff
@@ -0,0 +1,148 @@
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[contact\_form\] Contact Form](#-contact_form-contact-form)
+  - [🛠 Built With ](#-built-with-)
```

Comment:
You did a good job implementing the README file, but this section would need a little bit of fixing so you can stick to the template given  [here](https://github.com/microverseinc/readme-template) 💯 . If you notice, you have repeated `Table of content` and you wrongly placed the project name which does not tally with the template I tagged above. Your README is the first point of contact for your project, and its best to make it professional by sticking to the template 🔎
## Your Design:
![R2](https://user-images.githubusercontent.com/50990925/210999690-4434804e-74df-4206-a577-76b6695014fd.png)

Template Design:
![R1](https://user-images.githubusercontent.com/50990925/210999763-2184720e-a1c2-4fb3-a4eb-9c0743dc243d.png)